internal/local/command_kafka_start.go

```diff
@@ -115,91 +133,121 @@ func (c *Command) kafkaStart(cmd *cobra.Command, args []string) error {
 		return err
 	}
 	output.Println("\r")
-
 	log.CliLogger.Tracef("Successfully pulled Confluent Local image")
 
-	if err := c.prepareAndSaveLocalPorts(cmd, c.Config.IsTest); err != nil {
+	numBrokers, err := cmd.Flags().GetInt32("brokers")
```

Comment:
```suggestion
	brokers, err := cmd.Flags().GetInt32("brokers")
```

Reply:
want to rename this flag as "num-brokers"

Reply:
Up to you! Maybe check our other `int` and `int32` flags to see if there's a precendent?